### ISSUE TITLE
Crash on read rights for BUILTIN\Adminnistrator

### ISSUE THREAD

**Issue by macintoshplus** [InExtenso/win32service#87]
System: Windows 10
PHP Version: PHP 8.3.1 NTS X64
Extension version: 1.1.0-beta2
SAPI: Interactive shell

Code to reproduce:

```php
php > $info = win32_read_right_access_service('WindowsServicePhpTest', 'BUILTIN\Administrateurs');
php > var_dump($info->getRights());
```

Expected result: var_dump output of rights.

Actual result:

```
array(14) {
  [983551]=>
  string(18) "SERVICE_ALL_ACCESS"
  [2]=>
  string(21) "SERVICE_CHANGE_CONFIG"
  [8]=>
  string(28) "SERVICE_ENUMERATE_DEPENDENTS"
  [128]=>
  string(19) "SERVICE_INTERROGATE"
  [64]=>
  string(22) "SERVICE_PAUSE_CONTINUE"
  [1]=>
  string(20) "SERVICE_QUERY_CONFIG"
  [4]=>
  string(20) "SERVICE_QUERY_STATUS"
  [16]=>
  string(13) "SERVICE_START"
  [32]=>
  string(12) "SERVICE_STOP"
  [256]=>
  string(28) "SERVICE_USER_DEFINED_CONTROL"
  [65536]=>
  string(4047968968097231973) "
D:\via_phpize\php-8.3.1-nts-Win32-vs16-x64>echo %errorlevel%
-1073741819
```

**Comment by macintoshplus** [comment]
Request right without `BUILTIN\` domain work:

```php
$info = win32_read_right_access_service('WindowsServicePhpTest', 'Administrateurs');
var_dump($info->getRights());
```
```
array(14) {
  [983551]=>
  string(18) "SERVICE_ALL_ACCESS"
  [2]=>
  string(21) "SERVICE_CHANGE_CONFIG"
  [8]=>
  string(28) "SERVICE_ENUMERATE_DEPENDENTS"
  [128]=>
  string(19) "SERVICE_INTERROGATE"
  [64]=>
  string(22) "SERVICE_PAUSE_CONTINUE"
  [1]=>
  string(20) "SERVICE_QUERY_CONFIG"
  [4]=>
  string(20) "SERVICE_QUERY_STATUS"
  [16]=>
  string(13) "SERVICE_START"
  [32]=>
  string(12) "SERVICE_STOP"
  [256]=>
  string(28) "SERVICE_USER_DEFINED_CONTROL"
  [65536]=>
  string(6) "DELETE"
  [131072]=>
  string(12) "READ_CONTROL"
  [262144]=>
  string(9) "WRITE_DAC"
  [524288]=>
  string(11) "WRITE_OWNER"
}
```

**Comment by macintoshplus** [comment]
Fixed in v1.1.0beta3
